cross-make-mingw.sh

```diff
@@ -20,6 +20,12 @@ if [ "X$WINDRES" = "X" ]; then
     done
 fi
 
+for check in $CMD_PREFIX; do
+    if [[ ! $PATH == *"${check}"* ]]; then
+	export PATH="/usr/${check}:$PATH"
```

Comment:
Indentation

Reply:
Also what does this do?

Reply:
Yes, ive edited this on windows with vs code and it messed up the layout. Also this needs an extra check that i added later and didnt add to this PR yet. Will fix and comment.

This adds the mingw header files include directory to the PATH so mingw gcc can find the headers during the build.

Reply:
Fixed